config/index.js

```diff
@@ -2,21 +2,16 @@ import Config from 'react-native-config';
 
 const bitcoin = require('bitcoinjs-lib');
 
-const { SENTRY_DSN, HOST, PORT, BTCV_NETWORK, PROTOCOL, ELECTRUM_X_PROTOCOL_VERSION, IS_BETA } = Config;
-
-let isBeta = false;
-try {
-  isBeta = JSON.parse(IS_BETA);
-} catch (_) {}
+const { SENTRY_DSN, HOST, PORT, BTCV_NETWORK, PROTOCOL, ELECTRUM_X_PROTOCOL_VERSION, IS_BETA, APP_ID, APPLICATION_NAME } = Config;
 
```

Comment:
IMO this file should be removed at all in the next PR

Reply:
what do you want to use instead?

Reply:
Wherever it's required, simply import `'react-native-config'` and use the env variable directly

Reply:
I prefer to  have this file as it gives a level of abstraction, we can parse env vars here, add default one, and config that doesn't depend on env var

Reply:
Okay then, if you find it useful let's keep it. It actually makes sense 👍